app/maui-blazor/Platforms/MacCatalyst/AppDelegate.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Foundation;
+
+namespace MauiBlazor;
+
+[Register("AppDelegate")]
```

Comment:
Should this use `nameof`?

```suggestion
[Register(nameof(AppDelegate))]
```

Reply:
I'm not sure but these might not be semantically the same even though the text is the same. It's from the default MAUI project template so I think it's best to preserve it.

Reply:
Fixed.